pypyr/log/logger.py

```diff
@@ -19,6 +22,30 @@ def set_logging_config(log_level, handlers):
         handlers=handlers)
 
 
+class NoticeLogger(logging.getLoggerClass()):
```

Comment:
instead of the class, what I had in mind was something like this:
```
NOTIFY = 25


def notify(self, msg, *args, **kwargs):
    if self.isEnabledFor(NOTIFY):
        self._log(NOTIFY, msg, args, **kwargs)


def set_up_notify_log_level():
    logging.addLevelName(NOTIFY, "NOTIFY")
    logging.NOTIFY = NOTIFY
    logging.Logger.notify = notify
```

Reply:
Isn't subclassing from `Logger` more explicit and flexible? Monkey patching is the thing that I'm trying to avoid.

Reply:
I thought you might say something like that 😄 

yep, in all honesty, I'm in two minds about it. . . the monkey patch is a lighter touch intervention, but yes, you can make a very good argument that sub-classing is more the approved "OO" way. I sort of aimed for the monkey patch as a "lesser of two evils" sort of pick, but I don't feel like I can particularly justify it beyond reproach!